certbot-auto

```diff
@@ -781,20 +781,11 @@ elif [ -f /etc/redhat-release ]; then
   prev_le_python="$LE_PYTHON"
   unset LE_PYTHON
   DeterminePythonVersion "NOCRASH"
-  if [ "$PYVER" -eq 26 ]; then
```

Comment:
I don't think these lines should be changed. Unfortunately, certbot-auto has become pretty convoluted and difficult to work on over the years, but what's going on here is the code first determines your Python version without looking for Python 3. On RHEL 6 based systems, this will usually only find Python 2.6 which we no longer support and this branch will install Certbot using Python 3 from EPEL 6. The other branch installs Certbot using Python 2.7.